cranelift-codegen/src/ir/constant.rs

```diff
@@ -44,21 +47,23 @@ impl ConstantPoolEntry {
     }
 }
 
-/// Maintains the mapping between a constant handle (i.e.
-/// [`Constant`](crate::ir::Constant)) and its constant data (i.e.
-/// [`ConstantData`](crate::ir::ConstantData)).
+/// Maintains the mapping between a constant handle (i.e.  [`Constant`](crate::ir::Constant)) and
+/// its constant data (i.e.  [`ConstantData`](crate::ir::ConstantData)).
 #[derive(Clone)]
 pub struct ConstantPool {
-    /// This mapping maintains the insertion order as long as Constants are created with sequentially increasing integers.
+    /// This mapping maintains the insertion order as long as Constants are created with
```

Comment:
So should I be trying to keep comments under 100 characters?

Reply:
Yes please! :) It can't be enforced with the rustfmt right now because this is a Nightly feature (turns out, formatting comments containing code examples is complicated).

Reply:
Sounds good, I'll find my IDE setting for that.